Makefile

```diff
@@ -100,10 +88,14 @@ else
 KUSTOMIZE=$(shell which kustomize)
 endif
 
-generate-metallb-manifests:
+generate-metallb-manifests:  ## Generate metallb manifests
```

Comment:
Any reason why kustomize was not documented?

Reply:
kustomize is always embedded with other make target, you won't run it by itself mostly , plus this target set envvar not sure if that is something u need to see in the help ?

Reply:
That's the same for controller-gen (they are basically identical), I am fine on leaving both out / in as long as it's consistent